docker-compose.yml

```diff
@@ -174,6 +174,9 @@ services:
       GRAPHITE_PROTOCOL: PLAINTEXT
       CORE_SECRET_PROVIDER: secret-file
       CORE_SECRET_FILES: /mnt/secrets.properties
+      LOG_FILE_PROVIDER_NAME: loki
+      LOKI_SERVER_URL: "http://loki:3100"
+      LOKI_NAMESPACE: "compose"
```

Comment:
@Etsija you might be interested in this change.

Reply:
So, should this commit close https://github.com/eclipse-apoapsis/ort-server/issues/168?